crashappsec/chalk
# build: Update ARTIFACT_VERSION in autocomplete files

Updates the autocomplete files to reflect the most recent version

A future improvement could be to pull the version information in util.nim from the same place that the Makefile pulls it



- [ ] Updated `CHANGELOG.md` if necessary

## Issue

Chalk never updates the autocomplete files to insert the ARTIFACT_VERSION so continually replaces the file as it's used.
#251 


## Description

Updates util.nim and the autocomplete files with the correct version.
Also updates the Makefile with a new target to run the nimble task that will update the autocomplete files



## Testing


1) compile chalk
2) run any chalk command to install the new autocomplete file
3) run any chalk command to see that the autocomplete file is no longer copied

```
~/chalk ❯❯❯ ./chalk env
info:  Updating autocomplete script to version: 0.4.9
info:  Installed bash auto-completion file to: /home/numeral/.local/share/bash_completion/completions/chalk.bash
info:  /home/numeral/.local/chalk/chalk.log: Open (sink conf='default_out')
info:  Full chalk report appended to: ~/.local/chalk/chalk.log
[...]

~/chalk ❯❯❯ ./chalk env
info:  /home/numeral/.local/chalk/chalk.log: Open (sink conf='default_out')
info:  Full chalk report appended to: ~/.local/chalk/chalk.log
```
